app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      def show
+        volunteer = Volunteer.find(params[:id])
```

Comment:
Also, use the methods with the bang operator(!) such as `.find!`, use `.create!(object_params)` instead of `.new` and `save` and so on. The reason for this is I wrote a custom error handling module that returns json errors. You can find it in `app/controllers/concerns`, it rescues from all exceptions but methods without the bang operator do not trigger exceptions and we need them to so the appropriate JSON messages are returned to the client

Reply:
Done.